libs/langchain/langchain/llms/bedrock.py

```diff
@@ -80,7 +81,7 @@ def prepare_input(
         input_body = {**model_kwargs}
         if provider == "anthropic":
             input_body["prompt"] = _human_assistant_format(prompt)
-        elif provider == "ai21" or provider == "cohere":
+        elif provider == "ai21" or provider == "cohere" or provider == "meta":
```

Comment:
Suggesting a minor update to simplify the check. Accepting this should also kick in the CI. 
```suggestion
        elif provider in ["ai21", "cohere", "meta"]:
```